app/src/ereferrals/java/org/eyeseetea/malariacare/data/database/datasources/SettingsDataSource.java

```diff
@@ -109,6 +103,40 @@ private String getMediaPreference() {
                 "");
     }
 
+    private void saveProgramUrl(String programUrl) {
```

Comment:
Can you avoid duplicate code here? (PreferenceManager.getDefaultSharedPreferences)